src/lib/routes/admin-api/favorites.ts

```diff
@@ -34,6 +34,9 @@ export default class FavoritesController extends Controller {
                 openApiService.validPath({
                     tags: ['Features'],
                     operationId: 'addFavoriteFeature',
+                    summary: 'Add feature to favorites',
+                    description:
+                        'This endpoint marks the feature in the url as favorite',
                     responses: { 200: emptyResponse },
```

Comment:
This can probably return a 404 if the feature doesn't exist, right?  Also: this is a post, but I'm guessing it has no body, right?

```suggestion
                    responses: { 200: emptyResponse, ...getStandardResponses(401, 404) },
```